src/ixdat/readers/zilien.py

```diff
@@ -1,61 +1,314 @@
+"""Parsers for files produces by the Zilien software from Spectro Inlets"""
+
+import datetime
 import re
+import time
+from collections import defaultdict
+from pathlib import Path
+
 import pandas as pd
 import numpy as np
-from pathlib import Path
+from matplotlib import pyplot as plt
+
 from ..data_series import DataSeries, TimeSeries, ValueSeries, Field
 from ..techniques import ECMSMeasurement, MSMeasurement, ECMeasurement, Measurement
 from ..techniques.ms import MSSpectrum
 from .reading_tools import timestamp_string_to_tstamp, FLOAT_MATCH
-from .ec_ms_pkl import measurement_from_ec_ms_dataset
+
 
 ZILIEN_TIMESTAMP_FORM = "%Y-%m-%d %H_%M_%S"  # like 2021-03-15 18_50_10
```

Comment:
Zilien files get saved with a different timestamp form. For example: "2022_03_30_18_47_24_my_title.tsv"**
When that happens here I've asked my colleagues to change it to the normal timestamp format. But it would be nice if they didn't have to.

Reply:
Agreed. What's your opinion on this, should I try and account for it here, or merely log it as a olefn issue (which are now finally being worked on) ?

Reply:
I think it seems to only happen when the measurement has exited in an improper way.  So... fine to leave to leave it as is here.
Otherwise it'd just require inputting `forms` instead of `form` on line 141. I may do so in a patch if it comes up again here (hopefully not!).